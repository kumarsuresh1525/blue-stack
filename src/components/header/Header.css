.header {
  background-color: #1F2640;
  height: 80px; }
  .header a {
    color: #FFFFFF; }
  .header #language {
    width: 100px; }
