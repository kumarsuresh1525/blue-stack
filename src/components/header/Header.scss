@import '../../assets/styles/variables';

.header {
  background-color: #1F2640;
  height: 80px;

  a {
    color: $white;
  }

  #language {
    width: 100px;
  }
}