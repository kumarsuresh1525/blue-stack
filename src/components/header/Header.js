import React from 'react';
import {
  Navbar,
  NavbarBrand,
  Input,
  Container} from 'reactstrap';
import logo from '../../assets/images/logo.svg';
import './Header.css';

const Header = () => {
  return (
    <div>
      <Navbar className='header'>
        <Container>
          <NavbarBrand href="/"><img alt='logo' className='logo' src={logo}/> BlueStacks</NavbarBrand>
          <Input type="select" name="select" id="language">
            <option>English</option>
            <option>German</option>
          </Input>
        </Container>
      </Navbar>
    </div>
  )
}

export default Header;