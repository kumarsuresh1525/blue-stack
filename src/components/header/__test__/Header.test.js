import React from 'react';
import {render} from '@testing-library/react';
import Header from '../Header';

describe("Home Page test", () => {
  test("should render Home component", () => {
    const {container} = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});