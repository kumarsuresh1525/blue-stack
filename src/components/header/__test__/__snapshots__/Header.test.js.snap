// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page test should render Home component 1`] = `
<div>
  <div>
    <nav
      class="header navbar"
    >
      <div
        class="container"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          <img
            alt="logo"
            class="logo"
            src="logo.svg"
          />
           BlueStacks
        </a>
      </div>
    </nav>
  </div>
</div>
`;
